### DISCUSSION TITLE
I can't join GitHub education also iam a student

### DISCUSSION THREAD

**Discussion by ahmedhalo0l** [community/community#108095] [GitHub Education]
### Hi everyone,

I uploaded my ID but it said that it had uploaded previously but there is no one used it. is there is any one know where is the issue?

**Comment by mw-xyz** [comment]
Perhaps try using a different ID like a transcript. It is possible someone with a very similar looking ID to yours had already uploaded it, or there was a glitch. Lmk if this helps

**Comment by alsheroziy** [comment]
Hello everyone, I just joined the github community, please tell me the rules
